test/workers/test_websocket_worker.py

```diff
@@ -41,17 +41,77 @@ def test_websocket_workers_search(hook, start_proc):
     base_kwargs = {"id": "fed2", "host": "localhost", "port": 8766, "hook": hook}
     server_kwargs = base_kwargs
     server_kwargs["data"] = [sample_data]
-    server = start_proc(WebsocketServerWorker, server_kwargs)
+    process_remote_worker = start_proc(WebsocketServerWorker, server_kwargs)
 
     time.sleep(0.1)
 
-    client_worker = WebsocketClientWorker(**base_kwargs)
+    local_worker = WebsocketClientWorker(**base_kwargs)
 
     # Search for the tensor located on the server by using its tag
-    results = client_worker.search("#sample_data", "#another_tag")
+    results = local_worker.search("#sample_data", "#another_tag")
 
     assert results
     assert results[0].owner.id == "me"
     assert results[0].location.id == "fed2"
 
-    server.terminate()
+    local_worker.remove_worker_from_local_worker_registry()
+    process_remote_worker.terminate()
+
+
+def test_list_objects_remote(hook, start_proc):
+
+    kwargs = {"id": "fed", "host": "localhost", "port": 8765, "hook": hook}
+    process_remote_fed1 = start_proc(WebsocketServerWorker, kwargs)
+
+    time.sleep(1)
+
+    kwargs = {"id": "fed", "host": "localhost", "port": 8765, "hook": hook}
+    local_fed1 = WebsocketClientWorker(**kwargs)
+
+    x = torch.tensor([1, 2, 3]).send(local_fed1)
+
+    res = local_fed1.list_objects_remote()
+    res_dict = eval(res.replace("tensor", "torch.tensor"))
```

Comment:
Why do you need this eval?

Reply:
Because currently the result is a string. And eval transforms it into a dictionary. 
Currently this function is only used in the __str__ method of the websocket client worker. And there a string is used.